plugins/modules/cloud/misc/proxmox_template.py

```diff
@@ -216,6 +240,19 @@ def main():
             content_type = module.params['content_type']
             src = module.params['src']
 
+            # download appliance template
+            if content_type == 'vztmpl' and not src:
+                template = module.params['template']
+
+                if not template:
+                    module.fail_json(msg='template param to downloading appliance template is mandatory')
```

Comment:
English is not my mother tongue but shouldn't it be “for downloading”?

Reply:
you're right